static/app/views/organizationLayout/index.tsx

```diff
@@ -56,14 +57,16 @@ interface LayoutProps extends Props {
 function AppLayout({children, organization}: LayoutProps) {
   return (
     <AppContainer className="app">
-      <Nav />
-      {/* The `#main` selector is used to make the app content `inert` when an overlay is active */}
-      <BodyContainer id="main">
-        {organization && <OrganizationHeader organization={organization} />}
-        {organization && <DevToolInit />}
-        <Body>{children}</Body>
-        <Footer />
-      </BodyContainer>
+      <NavContextProvider>
```

Comment:
Wrapping the entire app in the `NavContextProvider` allows products to customize behavior based on the sidemenu state (needed for issues stream).